src/screen_capture/ScreenCapture.cpp

```diff
@@ -0,0 +1,226 @@
+#include <tuple>
+#include "ScreenCapture.h"
+
+namespace OutputCapture
+{
+    ScreenCapture::ScreenCapture()
+    {}
+
+    ScreenCapture::~ScreenCapture()
+    {
+        DeleteInstance();
+    }
+
+    ScreenCapture *OutputCapture::ScreenCapture::pInstance = nullptr;
+
+    bool ScreenCapture::CaptureFullScreen(std::string outputFileName) const
+    {
+        const std::tuple<uint16_t, uint16_t> screenSize = this->GetDesktopResolution();
+
+        return this->StartCapture(0, 0, std::get<0>(screenSize), std::get<1>(screenSize), outputFileName.c_str());
+    }
+
+    int ScreenCapture::GetFilePointer(HANDLE FileHandle) const
+    {
+        return SetFilePointer(FileHandle, 0, 0, FILE_CURRENT);
+    }
+
+    bool ScreenCapture::SaveBMPFile(const char *filename, HDC bitmapDC, uint16_t width, uint16_t height) const

```

Comment:
change to isSaveBMPFileSucessfull and create another function to save the file. (a bool function should always begin by a question, is..., does... etc..)